src/seedu/addressbook/parser/Parser.java

```diff
@@ -56,34 +71,36 @@ public Command parseCommand(String userInput) {
         final String commandWord = matcher.group("commandWord");
         final String arguments = matcher.group("arguments");
         switch (commandWord) {
+        case AddCommand.COMMAND_WORD:
+            return prepareAdd(arguments);
 
-            case AddCommand.COMMAND_WORD:
-                return prepareAdd(arguments);
+        case DeleteCommand.COMMAND_WORD:
+            return prepareDelete(arguments);
 
-            case DeleteCommand.COMMAND_WORD:
-                return prepareDelete(arguments);
+        case ClearCommand.COMMAND_WORD:
+            return new ClearCommand();
 
-            case ClearCommand.COMMAND_WORD:
-                return new ClearCommand();
+        case FindCommand.COMMAND_WORD:
+            return prepareFind(arguments);
 
-            case FindCommand.COMMAND_WORD:
-                return prepareFind(arguments);
+        case ListCommand.COMMAND_WORD:
+            return new ListCommand();
 
-            case ListCommand.COMMAND_WORD:
-                return new ListCommand();
+        case ListSortedCommand.COMMAND_WORD:
+            return new ListSortedCommand();
 
-            case ViewCommand.COMMAND_WORD:
-                return prepareView(arguments);
+        case ViewCommand.COMMAND_WORD:
+            return prepareView(arguments);
 
-            case ViewAllCommand.COMMAND_WORD:
-                return prepareViewAll(arguments);
+        case ViewAllCommand.COMMAND_WORD:
+            return prepareViewAll(arguments);
 
-            case ExitCommand.COMMAND_WORD:
-                return new ExitCommand();
+        case ExitCommand.COMMAND_WORD:
+            return new ExitCommand();
 
-            case HelpCommand.COMMAND_WORD: // Fallthrough
-            default:
-                return new HelpCommand();
+        case HelpCommand.COMMAND_WORD: // Fallthrough
+        default:
+            return new HelpCommand();
```

Comment:
It's good that you fixed this code layout problem. But note that a PR should contain only one independent change. Unrleated refactorings are best submitted as separate PRs.